src/commandline_frame.ts

```diff
@@ -189,8 +189,14 @@ commandline_state.clInput.addEventListener(
             // This is definitely a hack. Should expand aliases with exmode, etc.
             // but this whole thing should be scrapped soon, so whatever.
             if (response.value.startsWith("ex.")) {
-                const funcname = response.value.slice(3)
-                commandline_state.fns[funcname]()
+                const [funcname, ...args] = response.value.slice(3).split(/\s+/)
+
+                if (args.length === 0) {
+                    commandline_state.fns[funcname]()
+                } else {
+                    commandline_state.fns[funcname](args)
```

Comment:
```suggestion
                    commandline_state.fns[funcname](...args)
```

If I try to rebind `<C-o>yy` to this:

```patch
diff --git a/src/lib/config.ts b/src/lib/config.ts
index f26a0575..cd14523d 100644
--- a/src/lib/config.ts
+++ b/src/lib/config.ts
@@ -120,7 +120,7 @@ export class default_config {
         "<S-Tab>": "ex.prev_completion",
         "<Space>": "ex.insert_space_or_completion",

-        "<C-o>yy": "ex.copy_completion",
+        "<C-o>yy": "ex.execute_ex_on_completion clipboard yank",
     }

     /**
```

I get `# controller while accepting: Error: Not an excmd: clipboard,yank` when using it. 

I think this is probably the source of the problem but I haven't tested it :)

Reply:
Good catch.  This should be fixed in the latest commit.